hudi-hadoop-common/src/main/java/org/apache/hudi/io/hadoop/HoodieAvroOrcReader.java

```diff
@@ -58,7 +58,7 @@ public class HoodieAvroOrcReader extends HoodieAvroFileReader {
   public HoodieAvroOrcReader(HoodieStorage storage, StoragePath path) {
     this.storage = storage;
     this.path = path;
-    this.orcUtils = FileFormatUtils.getInstance(HoodieFileFormat.ORC);
+    this.orcUtils = new HoodieHadoopIOFactory(storage).getFileFormatUtils(HoodieFileFormat.ORC);
```

Comment:
same here, can we use `HoodieIOFactory.getIOFactory(storage).getFileFormatUtils(HoodieFileFormat.ORC)`?

Reply:
Fixed.